flows.json

```diff
@@ -1232,6 +1232,29 @@
             []
         ]
     },
+    {
+        "id": "4b6dba9f0b7031fc",
```

Comment:
![image](https://github.com/user-attachments/assets/39c2dae5-91a2-4ba0-91a9-0ac28d219118)
 Not needed I think. This test is done before the read file.

Reply:
needed when we delete image because we go to the init

Reply:
I don't see the relation between delete a image and connection with this catch node.

Reply:
fixed. Was for the journal.